dotcom-rendering/src/web/components/MostViewedFooterGrid.tsx

```diff
@@ -130,7 +130,7 @@ export const MostViewedFooterGrid = ({
 	const [selectedTabIndex, setSelectedTabIndex] = useState<number>(0);
 	return (
 		<>
-			{Array.isArray(data) && data.length > 1 && (
+			{Array.isArray(data) && (
```

Comment:
would `data` ever not be an array? If not, there seems something slightly odd about checking that it is, but also not (afaict) having any defined behaviour for cases where it's not an array? Not a big deal though I don't think